src/components/Incorporation/IncorporationDateTime.vue

```diff
@@ -423,7 +423,7 @@ label {
     top: -2rem;
     padding: 0 12px;
     font-size: $px-12;
-    font-weight: 500;
+    font-weight: normal;
```

Comment:
We only use "normal" and "bold", as declared in the font section of the base CSS.